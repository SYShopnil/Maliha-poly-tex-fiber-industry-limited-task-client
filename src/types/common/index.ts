export interface ICommonReturnData {
  message: string;
  status: number;
}

export enum EAuth {
  AuthTokenCookieName = "auth",
}

export enum EDataTestId {
  cLoginFormWithSubmit = "cLoginFormWithSubmit",
  cLogoutContainer = "cLogoutContainer",
  SHeaderMain = "SHeaderMain",
  SProfile = "SProfile",
}
