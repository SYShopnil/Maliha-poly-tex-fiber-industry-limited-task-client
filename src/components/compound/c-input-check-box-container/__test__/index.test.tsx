import { render, screen } from "@testing-library/react";
import { EDataTestId } from "@src/types/common";
import { CInputCheckBoxContainer } from "..";

describe("When Input Check Box Component Rendered", () => {
  it("Expect render successfully", () => {
    render(<CInputCheckBoxContainer />);
    const myElement = screen.getByRole(EDataTestId.CAddNewText);
    expect(myElement).toMatchSnapshot();
  });
});
