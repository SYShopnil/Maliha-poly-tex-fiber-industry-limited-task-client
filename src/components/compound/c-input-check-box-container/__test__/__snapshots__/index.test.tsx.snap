// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Input Check Box Component Rendered Expect render successfully 1`] = `
<div
  role="CAddNewText"
>
  <div>
    <div
      class="mb-4 grid grid-cols-12 gap-3"
    >
      <input
        class="px-2 py-1 border rounded mr-2 w-[100%] lg:col-span-8 col-span-4"
        type="number"
        value="0"
      />
      <button
        class="px-4 py-2 bg-blue-500 text-white rounded lg:col-span-4 col-span-8 lg:text-lg text-sm"
      >
        Add To Text boxes
      </button>
    </div>
  </div>
  <div>
    <div
      class="flex items-center my-2 space-x-4"
    >
      <input
        class="mr-2 h-[1.5rem] w-[1.5rem]"
        type="checkbox"
      />
      <div
        class="bg-[#FFE6CC]  w-[100%] flex justify-center item-center p-2"
      >
        <p
          class="text-lg font-bold "
        >
          All Select
        </p>
      </div>
    </div>
  </div>
  <div
    class="mt-4"
  >
    <div
      class="border-[2px] border-solid p-2 text-center bg-[#E1D5E7]"
    >
      <p>
        Selected all
         
        <strong>
          0
           
          Item
        </strong>
         and total number is
         
        <strong>
          0
        </strong>
      </p>
    </div>
    <div
      class="flex justify-end items-start mt-3"
    >
      <span
        class="cursor-pointer"
        role="CButton"
      >
        <span
          class="flex justify-center items-center  "
        >
          <span
            class="text-base md:text-base_2  flex justify-center items-center px-[1rem] py-[0.5rem] rounded-[0.563rem] w-[100%] h-[100%] bg-[#EDEDED] hover:bg-[#7A67EE]  duration-[0.5s] text-[#7A67EE] hover:text-[#EDEDED]   border-[0.063rem] hover:border-[#EDEDED] border-[#7A67EE]   "
          >
            <span />
          </span>
        </span>
      </span>
    </div>
  </div>
</div>
`;
